Jenkinsfile

```diff
@@ -0,0 +1,14 @@
+node("docker && awsaccess") {
+  cleanWs()
+  checkout scm
+
+  stage("Build Image") {
+    docker.build("ca_testing")
+  }
+
+  docker.image("ca_testing").inside {
+    stage("Lint and Unit Test") {
+      sh "bundle exec rake"
+    }
```

Comment:
Do you need to do the thing Zhelyan said in the software engineering channel?

```
withDockerRegistry(registry: [credentialsId: 'docker_hub']) {
    // code
}
```

Reply:
Probably haha. I'll add this!

Reply:
could you also remove the image when the tests are done?

```
stage("cleanup") { sh "docker rmi ca_testing -f || true" }
```